types/passport-saml/index.d.ts

```diff
@@ -97,5 +98,5 @@ export type Profile = {
 	getAssertion(): object;  // get the assertion XML parsed as a JavaScript object
 	getSamlResponseXml(): string; // get the raw SAML response XML
 } & {
-	[attributeName: string]: string;  // arbitrary `AttributeValue`s
+	[attributeName: string]: unknown;  // arbitrary `AttributeValue`s
```

Comment:
I don't believe this is correct. When the XML is parsed, the result is a `string`. I suppose it could be a `number`, but it certainly isn't `unknown`.

Reply:
@cjbarth If you want to expand the profile object, the current type forces the user to only have string values. For example, my Profile object has a `roles` attribute which is an array of string.

Reply:
@cjbarth Why would we force the string value for all attributeNames. `unknown` still allows for typing enforcement and allows for expansion of profile object.

Reply:
That is a good use-case, we probably need to update the docs then: https://github.com/bergie/passport-saml#configure-strategy-for-multiple-providers

Reply:
Could you make a PR to update the docs?

Reply:
@cjbarth https://github.com/bergie/passport-saml/pull/363. PR created.